pkg/controller/cluster_controller.go

```diff
@@ -58,7 +58,15 @@ type ClusterReconciler struct {
 // +kubebuilder:rbac:groups=policy,resources=poddisruptionbudgets,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=policy,resources=poddisruptionbudgets/status,verbs=get
 
+// Reconcile is the reconciliation loop entry point for cluster CRDs. It fetches the current cluster resources
+// and uses its state to interact with the world via a set of actions (i.e. init cluster, create a statefulset).
```

Comment:
nit: set of `Actors`?

Reply:
Yeah, naming is hard. So here is the thing. I called the field `Actions` in the controller struct. Those are assumed to be implemented by `Actors`. I used `actions` here as it is how it is referred in the current package. I've also extended the comment adding that detail.

Let me know if you still think it would be more clear.